spyder/plugins/history.py

```diff
@@ -230,6 +231,11 @@ def add_history(self, filename):
         editor.toggle_wrap_mode( self.get_option('wrap') )
 
         text, _ = encoding.read(filename)
+        linebreaks = [m.start() for m in re.finditer('\n', text)]

```

Comment:
We need to convert all EOL's to `\n` before doing this.

Reply:
Can't we just always use ``\n`` for our own history files, independent of the platform?